config/locales/ja.yml

```diff
@@ -1,15 +1,28 @@
 ---
 ja:
   about:
-    about_mastodon: Mastodon は<em>自由でオープンソース</em>なソーシャルネットワークです。商用プラットフォームの代替となる<em>分散型</em>を採用し、あなたのやりとりが一つの会社によって独占されるのを防ぎます。信頼できるインスタンスを選択してください &mdash; どのインスタンスを選んでも、誰とでもやりとりすることができます。 だれでも自分の Mastodon インスタンスを作ることができ、シームレスに<em>ソーシャルネットワーク</em>に参加できます。
+    about_mastodon_html: Mastodon は、オープンなウェブプロトコルを採用した、自由でオープンソースなソーシャルネットワークです。電子メールのように分散型の仕組みを採っています。
     about_this: このインスタンスについて
```

Comment:
"もっと詳しく" looks good on iPhone6 (375px), but it still needs more space on iPhone5s (320px).

Some thoughts:

* Reduce horizontal padding with media queries: e.g. 5px
* Using flex-grow to adjust padding: though it will also increase padding on wider devices
```
.landing-page .header .nav { display: flex; }
.landing-page .header .nav > * { flex: 1 1 auto; }
.landing-page .header .nav a { padding: 12px 0; }
```